publisher-sdk/src/main/java/com/criteo/publisher/CriteoInterstitial.java

```diff
@@ -98,6 +98,11 @@ public void loadAd(@Nullable BidToken bidToken) {
     }
   }
 
+  public void displayAd(@NonNull String signalData) {
+    getOrCreateController().notifyFor(CriteoListenerCode.VALID);
```

Comment:
Same than for banner.
Moreover, the display listener will take care of notifying publisher for success/failure of display

Reply:
Removed. It doesn't make sense indeed.